packages/now-client/src/utils/index.ts

```diff
@@ -143,17 +145,19 @@ export const prepareFiles = (
 
         if (options.isDirectory) {
           // Directory
-          fileName = options.path ? name.replace(`${options.path}/`, '') : name;
+          fileName = options.path
+            ? name.replace(isWin ? `${options.path}\\` : `${options.path}/`, '')
+            : name;
         } else {
           // Array of files or single file
-          const segments = name.split('/');
+          const segments = isWin ? name.split('\\') : name.split('/');
           fileName = segments[segments.length - 1];
         }
 
         next.push({
-          file: fileName,
+          file: fileName.replace(/\\/g, '/'),
```

Comment:
I'm not sure if we already have a normalizePath() function but we could probably add a common function since there are several packages that need this behavior (doesn't have to be in this PR).

Reply:
I think this normalization should only happen on Windows. In Unix, it's technically possible to have backslashes in a filename.

```
$ touch 'foo\bar'
$ ls -l foo\\bar 
-rw-r--r-- 1 nrajlich staff 0 Sep  6 14:28 'foo\bar'
```